views/includes/inset-text.html

```diff
@@ -1,3 +1,12 @@
-{{ govukInsetText({
-    html: headerBlock + paragraphOne + paragraphTwo
-}) }}
\ No newline at end of file
+{% set headerBlock = '<h2 class="govuk-heading-m">What information we’ll show on the public register</h2>' %}
```

Comment:
Could you please leave the inset text generic, it could be used for other reasons not just for the public register, and it could change anytime

Reply:
Not sure I can without repeating the code and causing sonar to complain.